Doc/tools/templates/customsourcelink.html

```diff
@@ -3,8 +3,14 @@
     <h3>{{ _('This Page') }}</h3>
     <ul class="this-page-menu">
       <li><a href="{{ pathto('bugs') }}">{% trans %}Report a Bug{% endtrans %}</a></li>
-      <li><a href="{{ pathto('_sources/' + sourcename, true)|e }}"
-            rel="nofollow">{{ _('Show Source') }}</a></li>
+      <li>
+        {% if version == "3.7" %}
```

Comment:
Should this updated when Python 3.7 is released?
I think 2.7, 3.5, and 3.6 branch should not have this if statement.
Or could we pass variable like `doc_source_url_prefix = "https://github.com/python/cpython/blob/3.6/Doc/"` from conf?